includes/admin/class-wc-rest-payments-settings-option-controller.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Class WC_REST_Payments_Settings_Option_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for settings options.
+ */
+class WC_REST_Payments_Settings_Option_Controller extends WC_Payments_REST_Controller {
+
+	/**
+	 * List of allowed option keys that can be updated via the REST API.
+	 *
+	 * @var array
+	 */
+	private const ALLOWED_OPTIONS = [
+		'wcpay_frt_discover_banner_settings',
```

Comment:
You can remove this one. `wcpay_frt_discover_banner_settings` was removed today in PR #10635.

Reply:
Addressed dc2a9037858b80240c17a15d5644219345beb4fe